blaze-server/src/main/scala/org/http4s/server/blaze/BlazeServer.scala

```diff
@@ -158,14 +162,14 @@ class BlazeBuilder(
 
           var lb = LeafBuilder(
             if (isHttp2Enabled) http2Stage(engine)
-            else http1Stage
+            else http1Stage(secure = true)
           )
           lb = prependIdleTimeout(lb)
           lb.prepend(new SSLStage(engine))
 
         case None =>
           if (isHttp2Enabled) logger.warn("HTTP/2 support requires TLS.")
```

Comment:
Off-topic for this PR, but should that warning say something about falling back to HTTP1?